raffles-bulletin-of-zoology.csl

```diff
@@ -0,0 +1,661 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+  <info>
+    <title>Raffles Bulletin of Zoology</title>
+    <title-short>RBZ</title-short>
+    <id>http://www.zotero.org/styles/raffles-bulletin-of-zoology</id>
+    <link href="http://www.zotero.org/styles/raffles-bulletin-of-zoology" rel="self"/>
```

Comment:
also, please add a template link like in line 7 with "template" with the style name for the style you based it on.

Reply:
is it something like this?
<link href="http://www.zotero.org/styles/apa" rel="template"/>

Reply:
yes